<testsuite name="default_suite_name" errors="0" failures="10" tests="21" time=".0075">
<testcase name="helloWorld_suite.testHelloWorld"/>
<testcase name="helloWorld_suite.testIgnoreWhitespaceHelloWorld"/>
<testcase name="myTests_suite.testAdd"/>
<testcase name="myTests_suite.testAddComplex"/>
<testcase name="myTests_suite.testAddComplexMultiDim"/>
<testcase name="myTests_suite.testAssociated"/>
<testcase name="myTests_suite.testNotAssociated"/>
<testcase name="myTests_suite.testAssociatedArrays"/>
<testcase name="myTests_suite.testWithParens"/>
<testcase name="Fixture_mod_suite.testRead"/>
<testcase name="Fixture_mod_suite.testEOF"/>
<testcase name="helloWorld_suite.testBrokenHelloWorld">
<failure message="Location: [helloWorld.pf:12], Intentionally broken test! String assertion failed:
    expected: ['Hello World!']
   but found: ['Found something else!']
  first diff:   ^
 "/>
</testcase>
<testcase name="myTests_suite.testAddBroken">
<failure message="Location: [myTests.pf:17], [[This test is intentionally broken.]] expected +1.000000 but found: +3.000000;  difference: |+2.000000| ] tolerance:+0.000000.
 "/>
</testcase>
<testcase name="myTests_suite.testAddComplexMultiDimBroken">
<failure message="Location: [myTests.pf:59], [[Intentionally broken.]] expected z=( +2.000000    , +0.000000    ) but found: z=( +0.000000    , +0.000000    );  difference: |+2.000000| ] tolerance:+0.000000;  first difference at element [2, 2].
 "/>
</testcase>
<testcase name="myTests_suite.testAssociatedBroken">
<failure message="Location: [myTests.pf:69], [[Intentionally broken.]] "/>
</testcase>
<testcase name="myTests_suite.testAssociatedWithBroken">
<failure message="Location: [myTests.pf:82], [[Intentionally broken.]] "/>
</testcase>
<testcase name="myTests_suite.testEqualityBroken">
<failure message="Location: [myTests.pf:147], [one] not equal to [two] "/>
</testcase>
<testcase name="myTests_suite.testEqualityWithMessageBroken">
<failure message="Location: [myTests.pf:167], [[Intentionally broken. 1==2.]] "/
 >
</testcase>
<testcase name="myTests_suite.testEquivalenceBroken">
<failure message="Location: [myTests.pf:179], [[YES,YES,YES]] not equal to [[YES,YES,NO]]
 "/>
</testcase>
<testcase name="myTests_suite.testEquivalenceWithMessageBroken">
<failure message="Location: [myTests.pf:200], [[Intentionally broken. YES.eqv.NO.]]
 "/>
</testcase>
<testcase name="myTests_suite.testWithParensBroken">
<failure message="Location: [myTests.pf:238], [[Intentionally broken, yes?]] "/>
</testcase>
</testsuite>
